wrappers/MathCAD/README.rst

```diff
@@ -1,20 +1,49 @@
-Wrapper of CoolProp for MathCAD 15
+CoolProp Wrapper for MathCAD 15
 ==================================
 
-copyright Scott Polak and Ian Bell, 2013
+| Copyright Scott Polak and Ian Bell, 2013. 
+| Updated by Jeff Henning, 2016.
 
-To Use
-======
+Prerequisites
+==============
+
+* You will need to have Visual Studio 2008 installed (Express version is fine).  Alternatively newer versions of Microsoft Visual Studio C++ should be fine.
```

Comment:
Yes, even the latest maintenance release, M045.  However, Prime is 64-bit.  This is the main difference between compiling for each version, even thought the header files are identical.